frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts

```diff
@@ -1102,6 +1153,56 @@ describe('Serving Runtime List', () => {
         expect(interceptions).to.have.length(1); // 1 dry-run request only
       });
     });
+
+    it('Successfully deletes model mesh model server', () => {
```

Comment:
I don't know if this actually results in a different workflow because the difference in models is simply an annotation.

All the same resources get deleted and I believe it's exactly the same modal.